src/modding/ModManager.tscn

```diff
@@ -1,598 +1,1053 @@
+__meta__ = {
+"_editor_description_": ""
+}
+
+[node name="RequiredModsButton" type="Button" parent="MainMarginContainer/VBoxContainer/InfoBoxSplit/ModInfoPanel/InfoBox/ModInfoContainer/MainModInfoContainer/InfoButtons/VBoxContainer/HBoxContainer"]
+margin_left = 218.0
+margin_right = 391.0
+margin_bottom = 35.0
+text = "REQUIRED_MODS"
+__meta__ = {
+"_editor_description_": ""
+}
 
-[node name="Url" type="Button" parent="CenterContainer/VBoxContainer/PanelContainer/MarginContainer/CurrentlySelected/ButtonsLine"]
-margin_left = 113.0
-margin_right = 262.0
-margin_bottom = 32.0
-custom_fonts/font = ExtResource( 11 )
-disabled = true
+[node name="IncompatibleButton" type="Button" parent="MainMarginContainer/VBoxContainer/InfoBoxSplit/ModInfoPanel/InfoBox/ModInfoContainer/MainModInfoContainer/InfoButtons/VBoxContainer"]
+margin_top = 39.0
+margin_right = 391.0
+margin_bottom = 74.0
+text = "INCOMPATIBLE_WITH"
+__meta__ = {
+"_editor_description_": ""
+}
+
+[node name="LoadOrderButton" type="Button" parent="MainMarginContainer/VBoxContainer/InfoBoxSplit/ModInfoPanel/InfoBox/ModInfoContainer/MainModInfoContainer/InfoButtons/VBoxContainer"]
+margin_top = 78.0
+margin_right = 391.0
+margin_bottom = 113.0
+text = "LOAD_ORDER"
+__meta__ = {
+"_editor_description_": ""
+}
+
+[node name="ModUrl" type="Button" parent="MainMarginContainer/VBoxContainer/InfoBoxSplit/ModInfoPanel/InfoBox/ModInfoContainer/MainModInfoContainer/InfoButtons/VBoxContainer"]
+margin_top = 117.0
+margin_right = 391.0
+margin_bottom = 152.0
 text = "OPEN_MOD_URL"
+__meta__ = {
+"_editor_description_": ""
+}
 
-[node name="Spacer2" type="Control" parent="CenterContainer/VBoxContainer"]
-margin_top = 489.0
-margin_right = 850.0
-margin_bottom = 494.0
-rect_min_size = Vector2( 0, 5 )
+[node name="MoreInfo" type="Button" parent="MainMarginContainer/VBoxContainer/InfoBoxSplit/ModInfoPanel/InfoBox/ModInfoContainer/MainModInfoContainer/InfoButtons/VBoxContainer"]
+margin_top = 156.0
+margin_right = 391.0
+margin_bottom = 191.0
+text = "MORE_INFO"
+__meta__ = {
+"_editor_description_": ""
+}
 
-[node name="HBoxContainer2" type="HBoxContainer" parent="CenterContainer/VBoxContainer"]
-margin_top = 498.0
-margin_right = 850.0
-margin_bottom = 530.0
-size_flags_horizontal = 3
-alignment = 2
+[node name="DescriptionSeperator" type="HSeparator" parent="MainMarginContainer/VBoxContainer/InfoBoxSplit/ModInfoPanel/InfoBox/ModInfoContainer"]
+margin_top = 201.0
+margin_right = 1415.0
+margin_bottom = 206.0
+custom_constants/separation = 5
+custom_styles/separator = SubResource( 3 )
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="Workshop" type="Button" parent="CenterContainer/VBoxContainer/HBoxContainer2"]
-margin_left = 517.0
-margin_right = 710.0
-margin_bottom = 32.0
-custom_fonts/font = ExtResource( 11 )
-text = "BROWSE_WORKSHOP"
+[node name="ModInfoDescription" type="RichTextLabel" parent="MainMarginContainer/VBoxContainer/InfoBoxSplit/ModInfoPanel/InfoBox/ModInfoContainer"]
```

Comment:
Our CustomRichTextLabel should be used here, it supports additional thrive bbcodes which might be used by modders for their mod descriptions.

Reply:
Yeah i been meaning to do this

Reply:
is there anything i have to do beside dragging the scene in to make it work?

Reply:
You then just simply need to use the `ExtendedBbcode` property to get and set the text instead of the built in BbcodeText property.